pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -763,11 +768,25 @@ export class CreateVmAction extends React.Component {
     }
 
     render() {
+        const createButton = (
+            <Button disabled={!this.props.systemInfo.osInfoList || !this.state.virtInstallAvailable} className="pull-right" id="create-new-vm" bsStyle='default' onClick={this.open} >
+                {_("Create VM")}
+            </Button>
+        );
+        let createButtonWrapper;
+
+        if (!this.state.virtInstallAvailable)
+            createButtonWrapper = (
```

Comment:
For simplicity you could also just reassign createButton here (and make it let instead of const).